packages/spark-extras/components/highlight-board/_highlight-board.scss

```diff
@@ -2,33 +2,58 @@
 // Highlight Board
 // ==================================================================
 
-// TODO: there are z-index issues in chrome on android
-
 //
 // Available Variables
 ///
-$highlight-board-content-padding: 1.949rem 1rem;
+$highlight-board-type-breakpoint: 30rem;
+$highlight-board-type-reduction-percentage: 0.8;
+$highlight-board-height: 31.25rem;
 
 .sprk-c-HighlightBoard {
+  background-color: $gray-light;
   text-align: center;
   position: relative;
 }
 
+.sprk-c-HighlightBoard--has-image {
+  background-color: $white;
+}
+
 .sprk-c-HighlightBoard__image {
-  height: 100%;
   object-fit: cover;
+  font-family: 'object-fit: cover;'; // required by the polyfill
+  height: $highlight-board-height;
+  width: 100%;
 }
 
 .sprk-c-HighlightBoard__content {
-  padding-top: 1.949rem;
-  padding-bottom: 1.949rem;
-  padding-left: 1rem;
-  padding-right: 1rem;
+  padding: $space-l $space-m;
```

Comment:
Could try using `$space-inset-tall-m: 24px 16px 24px 16px;` to see if it works out. That away we stay consistent if we can.